adam-r/bdgenomics.adam/tests/testthat/test_alignmentRecordRdd.R

```diff
@@ -105,3 +105,23 @@ test_that("transmute to coverage", {
     expect_true(is(readsAsCoverage, "CoverageRDD"))
     expect_equal(count(toDF(readsAsCoverage)), 5)
 })
+
+test_that("reads can cache", {
+
+    readsPath <- resourceFile("unsorted.sam")
+    reads <- loadAlignments(ac, readsPath)
+
+    cache(reads)
```

Comment:
Ditto RE these methods. What are we testing here?

Reply:
I made a better check for the python case but I am very unfamiliar with the R interface. How could I access the sc functions from the test (ie sc.getPersistentRDDs())?